File: FirstBit Workspace/C Programming/Assignments/Assignment 9/Q3.c
//Find sum of all numbers.
#include<stdio.h>
void sumOfElements(int*, int);
void arrayInputOutput(int*, int);

void main(){
	
	int arr[10];
	
	arrayInputOutput(arr, 10);
	sumOfElements(arr,10);
		
}	

void arrayInputOutput(int* ptr, int size){
	
	//storing numbers in a array
	printf("Enter 10 numbers to store in a array: ");
	for(int i=0;i<size;i++)
		scanf("%d",&ptr[i]);
		
	printf("\nYour array: ");
	for(int i=0;i<size;i++)
		printf("%d ",ptr[i]);
}

void sumOfElements(int* ptr, int size)
{
	
	int sum=0;
		
	printf("\nSum or all the elements: ");
	for(int i=0;i<size;i++){
		sum=sum+ptr[i];
	}
	printf("%d",sum);
		
}

File: FirstBit Workspace/Core Java/In Class Work/ArrayCProgramsInJava/src/Q3.java
import java.util.Scanner;

public class Q3 {

	public static void main(String[] args) {
		
		//Find sum of all numbers.
		
		//array reference 
		int[] array;
		
		//initialization
		array = new int[5];
		
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Enter elements to store in array: ");
		
		//store elements
		for(int i=0; i<array.length; i++) {
			array[i] = sc.nextInt(); 
		}
		
		System.out.println("\nElements stored in array: ");
		//display array
		for(int ele:array) {
			System.out.println(ele);
		}
		
		int sum = 0;
		
		for(int ele:array) {
			sum += ele;
		}
		
		System.out.println("\nSum of all elements: " + sum);
		
		sc.close();

	}

}
